sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/microsoft/opentelemetry/exporter/azuremonitor/export/__init__.py

```diff
@@ -47,9 +46,8 @@ def __init__(self, **options):
         default_storage_path = os.path.join(
             tempfile.gettempdir(), TEMPDIR_PREFIX + temp_suffix
         )
-        retry_policy = RetryPolicy(timeout=self._timeout)
         self.client = AzureMonitorClient(
-            parsed_connection_string.endpoint, retry_policy=retry_policy)
+            host=parsed_connection_string.endpoint, connection_timeout=self._timeout, **kwargs)
```

Comment:
Which generated class is using `connection_timeout`? How come we aren't using `RetryPolicy` anymore?

Reply:
Timeout is about the Http connection timeout so we need to use ConenctionConfiguration used by pipeline transport

https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/azure/core/configuration.py

Reply:
Which is the class that instantiates a `ConnectionConfiguration`?

Reply:
Also, should we make the default timeout `300` seconds to match the default in `ConenctionConfiguration `?

Reply:
The pipeline transport class will use that configuration, by default RequestsTransport

https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/azure/core/pipeline/transport/_requests_basic.py#L198